cmd/argocd/commands/headless/headless.go

```diff
@@ -237,7 +238,14 @@ func MaybeStartLocalServer(ctx context.Context, clientOpts *apiclient.ClientOpti
 		return fmt.Errorf("error creating kubernetes dynamic clientset: %w", err)
 	}
 
-	controllerClientset, err := client.New(restConfig, client.Options{})
+	scheme := runtime.NewScheme()
+	err = v1alpha1.AddToScheme(scheme)
```

Comment:
Why do we need to do this? This seems like something that should be done automatically on some init.

Reply:
It's needed in the headless mode, when login with the --core option.
On this case you use this local api server and not the real remote one.
It's not required in the regular mode